app/src/main/java/no/usn/mob3000/domain/model/content/NewsStruct.kt

```diff
@@ -0,0 +1,30 @@
+/**
+ * These are news related data structures needed for the UI layer.
+ *
+ * TODO: Abstraction, especially for "UpateData"
+ *
+ * @author 258030
+ * @created 2024-10-30
+ */
+
+package no.usn.mob3000.domain.model.content
+
+import kotlinx.datetime.Instant
+
+data class NewsData(
+    val newsId: String,
+    val createdAt: Instant,
+    val modifiedAt: Instant,
+    val createdByUser: String,
+    val title: String,
+    val summary: String,
+    val content: String,
+    val isPublished: Boolean
+)
+
+data class NewsUpdateData(
```

Comment:
Working with data structures that don't represent the database tables, shouldn't make it necessary to have different structs for "updated" data. An object should always represent its latest state, and so, the updated data should just be written to the `NewsData` struct instead.